backend/market/views.py

```diff
@@ -57,167 +68,163 @@ def get_queryset(self):
         return Offer.objects.filter(user=user)
 
 
-class Properties(viewsets.ModelViewSet):
+def apply_filters(
```

Comment:
I think passing this should be called create_filters and we shouldn't be passing in a queryset.
Also filter_mappings, is_sublet, and tag_field is a bit contrived. 

You should 
1. Make a (static) function that does this for Items and does it cleanly and well without extra accessories (in the Item view)
2. Make a function that does this cleanly and well for sublet (in the sublet view) that uses the Item function for help to make it more concise